materials_io/utils/interface.py

```diff
@@ -59,6 +71,24 @@ def get_available_adapters() -> dict:
     return _output_plugin_info(ExtensionManager(namespace='materialsio.adapter'))
 
 
+def get_adapter_map(adapter_map: str, parsers: list) -> dict:
+    """ Create reusable map of all available adapters.
+    Returns:
```

Comment:
Incomplete documentation. 

Also, why is this map "resuable"? Also, what does this map do? It's unclear on why someone would call this function unless they read this source code

Reply:
Changed this to invisible helper function _get_adapter_map. Documented it to better state its purpose (now that I admittedly understand it better than I did then)

Reply:
Could you add what the "adapter map" is: A mapping between a list of parsers and names of the appropriate adapters to use to format their output?